teuthology/contextutil.py

```diff
@@ -139,7 +139,6 @@ def __call__(self):
         if self.timeout > 0:
             self.sleep_current = min(self.timeout - self.total_seconds, self.sleep_current)
         self.total_seconds += self.sleep_current
-        print(self.total_seconds, self.sleep_current)
```

Comment:
Why was this piece removed?

Reply:
It was what appears to be a debug statement left over from development in the commit mentioned in my commit message. Results in random integers being printed to the console from who-knows-where